relays/client-substrate/src/headers_source.rs

```diff
@@ -16,93 +16,34 @@
 
 //! Default generic implementation of headers source for basic Substrate client.
 
-use crate::{
-	chain::{BlockWithJustification, Chain},
-	client::Client,
-	error::Error,
-};
+use crate::{chain::Chain, client::Client, error::Error};
```

Comment:
If I'm not missing anything, this file isn't used anymore. If may be deleted as well

Reply:
right! done.